kumascript/macros/accessibilitySidebar.ejs

```diff
@@ -0,0 +1,168 @@
+<%
+
+const locale = env.locale;
+const sidebarURL = `/docs/Web/`;
+const baseURL = `/${locale}/docs/Web/Accessibility/`;
+const AriaURL = `/${locale}/docs/Web/Accessibility/ARIA/`;
+const LearnURL = `/${locale}/docs/Learn/Accessibility/`;
+
+const l10nStrings = mdn.localStringMap({
+  "en-US": {
+    "Accessibility" : "Accessibility",
+    "Guides" : "Guides",
+        "Information_for_Web_authors" : "Information for web authors",
+        "Accessibility:_What_users_can_to_to_browse_safely" : "User settings",
+        "An_overview_of_accessible_web_applications_and_widgets" : "Applications and widgets",
+        "Keyboard-navigable_JavaScript_widgets" : "Keyboard-navigable widgets",
+        "Mobile_accessibility_checklist" : "Mobile checklist",
+        "Understanding_WCAG" : "Understanding WCAG",
+        "Cognitive_accessibility" : "Cognitive accessibility",
+        "Accessibility_and_Spacial_Patterns" : "Spacial Patterns",
+        "Understanding_Colors_and_Luminance" : "Colors and Luminance",
+        "Seizure_disorders" : "Seizures and physical reactions",
+    "Resources" : "Resources",
+        "What_is_accessibility" : "What is accessibility?",
+        "HTML" : "HTML: A good basis for accessibility",
+        "CSS_and_JavaScript" : "CSS and JavaScript accessibility best practices",
+        "WAI-ARIA_basics" : "WAI-ARIA basics",
+        "Multimedia" : "Accessible multimedia",
+        "Mobile" : "Mobile accessibility",
+    "ARIA" : "ARIA",
+    "ARIA_guides" : "ARIA guides",
+        "Annotations" : "Annotations",
+        "ARIA_Guides" : "Guides",
+        "ARIA_Live_Regions" : "Live regions",
+        "ARIA_Screen_Reader_Implementors_Guide" : "Screen reader implementors guide",
+        "ARIA_Techniques" : "Using ARIA",
+        "Multipart_labels" : "Multipart labels",
+        "How_to_file_ARIA-related_bugs" : "How to file bugs",
+    "ARIA states and properties": "ARIA states and properties",
+    "ARIA roles": "ARIA roles",
+  }
+});
+
+const sections = [
+  {
+    name: "Accessibility",
+    link: "Accessibility",
+    subsections: [
+      {
+        name: "Guides",
+        url: baseURL,
+        pages: [
+          "Information_for_Web_authors",
+          "Accessibility:_What_users_can_to_to_browse_safely",
+          "An_overview_of_accessible_web_applications_and_widgets",
+          "Keyboard-navigable_JavaScript_widgets",
+          "Mobile_accessibility_checklist",
+          "Understanding_WCAG",
+          "Cognitive_accessibility",
+          "Accessibility_and_Spacial_Patterns",
+          "Understanding_Colors_and_Luminance",
+          "Seizure_disorders",
+        ]
```

Comment:
What do you think about using "fully-qualified" slugs (without locale) here?

I think this explicit approach would have two advantages:

1. It's straight-forward in which section a new page has to be added.
2. It makes it easier to find where a page is referenced in the sidebar.

The disadvantage is that this would allow putting a page where it doesn't belong.

```suggestion
        pages: [
          "Web/Accessibility/Information_for_Web_authors",
          "Web/Accessibility/Accessibility:_What_users_can_to_to_browse_safely",
          "Web/Accessibility/An_overview_of_accessible_web_applications_and_widgets",
          "Web/Accessibility/Keyboard-navigable_JavaScript_widgets",
          "Web/Accessibility/Mobile_accessibility_checklist",
          "Web/Accessibility/Understanding_WCAG",
          "Web/Accessibility/Cognitive_accessibility",
          "Web/Accessibility/Accessibility_and_Spacial_Patterns",
          "Web/Accessibility/Understanding_Colors_and_Luminance",
          "Web/Accessibility/Seizure_disorders",
        ]
```